layer/store/wallet/authz.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  MsgGrant,
+  PrivateKey,
+  getEthereumAddress,
+  getDefaultSubaccountId,
+  getGenericAuthorizationFromMessageType
+} from '@injectivelabs/sdk-ts'
+import { Wallet } from '@injectivelabs/wallet-ts'
+import { GrantDirection, AutoSign, EventBus } from '../../types'
+import { useSharedWalletStore } from '.'
```

Comment:
Remove this

Reply:
i added this because we dont have types safety. its not autoimported... if i remove it we may run into bugs.. wyt?

Reply:
sure